src/unit-test/InputForm.test.js

```diff
@@ -15,32 +15,47 @@ it('render a form', () => {
 });
 
 it('render 3 text boxes', () => {
-  expect(wrapper.find({ name: "ing1" })).toHaveLength(1);
-  expect(wrapper.find({ name: "ing2" })).toHaveLength(1);
-  expect(wrapper.find({ name: "ing3" })).toHaveLength(1);
+  expect(wrapper.find({ name: "ing" })).toHaveLength(1);
 });
 
 it('render 1 submit button', () => {
   expect(wrapper.find({ type: "submit" })).toHaveLength(1);
 });
 
 it('renders without crashing', () => {
-  expect(wrapper.text()).toEqual("Ingredient 1: Ingredient 2:Ingredient 3:");
+  expect(wrapper.text()).toEqual("Ingredients:Add");
 });
 
 it('call function onHandleChange on text field change', () => {
   const onHandleChangeMock = jest.fn()
   const wrapper = shallow(<InputForm onHandleChange={onHandleChangeMock}/>)
-  const input1 = wrapper.find({ name: "ing1" })
+  const input1 = wrapper.find({ name:"ing" })
   input1.simulate('change')
   expect(onHandleChangeMock).toHaveBeenCalled()
 });
 
 it('call function onHandleSubmit on submit', () => {
   const onHandleSubmitMock = jest.fn()
   const wrapper = shallow(<InputForm onHandleSubmit={onHandleSubmitMock}/>)
-  const input1 = wrapper.find('form')
+  const input1 = wrapper.find({ type:'submit' })
   const eventMock = {preventDefault:function(){}}
-  input1.simulate('submit', eventMock)
+  input1.simulate('click', eventMock)
   expect(onHandleSubmitMock).toBeCalled()
 });
+
+it('call function handleSubmit on submit', () => {
```

Comment:
Not sure if this tests anything and might be redundant?

Reply:
Resolved, it was redundant